x-pack/metricbeat/module/aws/aws.go

```diff
@@ -36,6 +36,7 @@ type Config struct {
 	AWSConfig             awscommon.ConfigAWS `config:",inline"`
 	TagsFilter            []Tag               `config:"tags_filter"`
 	IncludeLinkedAccounts *bool               `config:"include_linked_accounts"`
+	LimitRestAPI          *int32              `config:"limit_rest_api"`
```

Comment:
Can we add some documentatino on what's this new config parameter LimitRestAPI please? Thanks!!

Reply:
Sure , I added [this](https://github.com/elastic/beats/pull/40755/files#diff-a43aae747ff228e3bf61ee6f27e2898587db11b4ae37286304ecb6d58181ceb7) and a comment [here](https://github.com/elastic/beats/pull/40755/files#diff-bafb9a6c22e4a9b94bfb65329d4cf4035b0664e5781192f2ac8a27f64a2c843dR109)

Reply:
Thanks!! Now I wonder if we should even expose this parameter to the users. Can we just use the default or the maximum value hardcoded instead? 
@zmoog WDYT?

Reply:
If we plan to keep this option, then we should mention the accepted value range and may be even validate the input to match with restrictions (ex:- max 500)

Reply:
Added a check here https://github.com/elastic/beats/pull/40755/commits/26a4375ba45c4cb8f35480c75cf8e6afb342637c